MoreLinq/Fold.g.tt

```diff
@@ -57,6 +57,9 @@ namespace MoreLinq
         /// <param name="source">The sequence of items to fold.</param>
         /// <param name="folder">Function to apply to the elements in the sequence.</param>
         /// <returns>The folded value returned by <paramref name="folder"/>.</returns>
+        /// <exception cref="ArgumentNullException"><paramref name="source"/> is null</exception>
+        /// <exception cref="ArgumentNullException"><paramref name="folder"/> is null</exception>
+        /// <exception cref="InvalidOperationException"><paramref name="source"/> does not contain exactly <#= e.CountElements #> elements</exception>
```

Comment:
`elements` is extra in `<#= e.CountElements #> elements` & is causing generated documentation to contain `1 elements elements`, `2 elements elements` and so on.